src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.tsx

```diff
@@ -27,6 +27,7 @@ import "./VenueRoomsEditor.scss";
 
 const styles: React.CSSProperties = {
   width: "100%",
+  height: "100%",
```

Comment:
[Q]
Can this be moved entirely in `VenueRoomsEditor.scss` or partially to allow something like:
```tsx
    <div ref={drop} style={{ ...styles, ...containerStyle }}>
```
to be replaced with something like:
```tsx
  const containerClasses = classNames(
    "VenueRoomsEditor",
    useCss(containerStyle ?? {})
  );

  return (
    <div ref={drop} className={containerClasses}>
```

[OOS]
At the end of the day, parent components shouldn't be passing class names to children but deal with child styles in their own SCSS files so it'd be good if there is no `containerStyle` prop to deal with.

Reply:
fixed in a4aa1c4f4ec90d75594111a0ed9e49045dfe7cd6